instagram/src/components/Post.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import CommentSection from './CommentSection';
+
+class Post extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { }
```

Comment:
Add likes to state here. Reference the props you've passed in.
```suggestion
        this.state = {
        likes: {this.props.likes}
        }
```